stable/fluent-bit/README.md

```diff
@@ -81,9 +81,12 @@ The following table lists the configurable parameters of the Fluent-Bit chart an
 | `podLabels`                        | Optional pod labels                       | `NULL`                |
 | `fullConfigMap`                    | User has provided entire config (parsers + system)  | `false`      |
 | `existingConfigMap`                | ConfigMap override                         | ``                    |
-| `extraEntries.input`               |    Extra entries for existing [INPUT] section                     | ``                    |
-| `extraEntries.filter`              |    Extra entries for existing [FILTER] section                     | ``                    |
-| `extraEntries.output`              |   Extra entries for existing [OUPUT] section                     | ``                    |
+| `extraEntries.kubernetes_input`    |    Extra entries for existing Kubernetes [INPUT] section           | ``                    |
+| `extraEntries.systemd_input`       |    Extra entries for the optional systemd [INPUT] section          | ``                    |
+| `extraEntries.kubernetes_input`    |    Extra entries for existing Kubernetes [INPUT] section           | ``                    |
+| `extraEntries.kubernetes_filter`   |    Extra entries for existing Kubernetes [FILTER] section          | ``                    |
+| `extraEntries.kmsg_filter`         |    Extra entries for the optional kmsg [FILTER] section            | ``                    |
+| `extraEntries.output`              |    Extra entries for existing [OUPUT] section                      | ``                    |
```

Comment:
really not sure how I feel about this. This is by far the cleanest. But is also a breaking change. Part of me wants to just leave all the kubernetes ones called the way they were and just add new ones.

Small it, for consistency, it should be camel cased not underscored.

If we do go with renaming things, we need some kind of note explaining how to migrate.